spec/stack.test.js

```diff
@@ -16,9 +16,73 @@ describe('Stack', () => {
   context('push()', () => {
     it('pushes an element to the top of the stack.', () => {
```

Comment:
Consider making this two separate tests:
1. Did the size of the stack change?
1. Did the correct element get added to the correct location?